approach.md

```diff
@@ -68,23 +70,23 @@ Partners will be a lot more invested in what you recommend if they understand an
 
 There are a few primary ways we do this:
 
-- **Meet with the partner regularly** (at least weekly) to discuss what you're working on, share insights, and ask questions. Get clear early on about what meeting tools will work for everyone, and use video calls if possible.
-- **Send a "Weekly Ship:"** a quick report emailed to the client at the end of each week. Here's the [Weekly Ship template](https://docs.google.com/document/d/1ulPzFvBLunXm3MD0xL8nPzlJDOoFJbNLPo0Sl6ph99Q/edit), an example of a [Weekly Ship doc](https://docs.google.com/document/d/1uWNqM_2I3i1-UAysYxXt9C3UN_892sJoeeUV0z6wPhM/edit#heading=h.lg378il4ug92), and [the Weekly Ship origin story](https://docs.google.com/document/d/1tdbh_rmIsRgMEBQXh5x6fVAAE4ajUHj8eZ2eFF4acO0/edit#). We also share Weekly Ships in #the-shipping-news Slack channel, so we can see what's happening across projects.
+- **Meet with the partner regularly** (at least weekly) to discuss what you're working on, share insights, and ask questions. Get clear early on about what meeting tools will work for everyone, and use video calls if possible. Include partners in stand-ups.
```

Comment:
Include partners in stand-ups is such a good tip, it makes me want to spool up another section on modelling agile practices and human-centered design. No change needed, just thinking out loud.